resources/crds/ripsaw_v1alpha1_smallfile_cr.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: example-benchmark
+  namespace: ripsaw
+spec:
+  workload:
+    name: smallfile
+    args:
+      clients: 0
+      operation: []
+      threads: 5
+      file_size: 64
+      files: 100
+      storageclass:
```

Comment:
are these parameters supposed to be empty? that is the value null?

Reply:
Fixed that , they are not supposed to be empty